pkg/controller/control.go

```diff
@@ -1170,6 +1179,82 @@ func (c *Controller) GetLatestMetics() *ptypes.Metrics {
 	return metrics
 }
 
+func (c *Controller) ShowProfiler() (string, bool, error) {
+	c.profilerLock.Lock()
+	defer c.profilerLock.Unlock()
+
+	logrus.Infof("Prepare to do ShowProfilerConfig")
+	if c.profilerServer != nil {
+		return c.profilerServer.Addr, true, nil
+	}
+	return "", true, nil
+}
+
+func (c *Controller) EnableProfiler(portNumber int32) (string, bool, error) {
+	c.profilerLock.Lock()
+	defer c.profilerLock.Unlock()
+
+	logrus.Infof("Prepare to do EnableProfiler")
+	var err error
+	if c.profilerServer != nil {
```

Comment:
Do we need to return an error or do anything if the input port number does not match the existing server port?

Reply:
Hi @shuo-wu, I do not understand that.

The `--port` argument lets the user define the port the profiler server was running. (It doesn't require)
So, in general, we assume that the user who uses `--port` will use a different port with the default port number.

Could you explain more about it?

Reply:
For example, the caller sends 2 profiler enabling request with different port numbers for an engine, then the server will return successes for both requests. But actually, the input port number of the 2nd request does not match the running profiler port

Reply:
Ah.. I know your means.
Indeed, this case should have a clear output.
Thanks!!  I will update that later.